Quaver.Shared/Quaver.Shared.csproj

```diff
@@ -7,6 +7,7 @@
     <DefineConstants>TRACE;DEBUG;VISUAL_TESTS</DefineConstants>
   </PropertyGroup>
   <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
I specifically asked @AiAe whether to use the more verbose approach or just use unsafe. I was told to use unsafe because it is more readable. I can change it back if you two come to an agreement.